### PR TITLE
Website clone

### PR DESCRIPTION
**Pull request by rehanvrana**
Working on Website clone and till now completed three components: navbar, slider and services

### CODE REVIEW

File: src/components/NavBar/Hamburger.tsx [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+              </Accordion.Title>
+              <Accordion.Content className="p-0">
+                <ul className="flex flex-col gap-4">
+                  {[
+                    ["Full width Gallery"],
+                    ["Grid Gallery"],
+                    ["Full width Masonry Gallery"],
+                    ["Masonry Gallery"],
+                  ].map(([item]) => (
+                    <li className="w-full p-2 px-4 bg-white hover:bg-[#756aee] transition-colors duration-300 ease-linear">
```
Comment (JaydeepParmar-iView) [inline src/components/NavBar/Hamburger.tsx]: make a variable and assign these values in it then map the variable here

File: src/components/NavBar/Hamburger.tsx [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+                    ["Service 2"],
+                    ["Pricing"],
+                    ["Single Service"],
+                    ["Coming Soon"],
+                    ["Our Team"],
+                    ["Contact Us 2"],
+                    ["Testimonials"],
+                    ["FAQ"],
+                    ["404 Page"],
+                    ["Search Results"],
```
Comment (JaydeepParmar-iView) [inline src/components/NavBar/Hamburger.tsx]: make a variable and then map

File: src/components/NavBar/Hamburger.tsx [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+                <div className="text-[#756aee] text-xl p-2 m-2 border-b">
+                  Elements
+                </div>
+                <ul className="flex flex-col gap-3">
+                  {[
+                    ["Typography"],
+                    ["Calls to Action"],
+                    ["Icons with Text"],
+                    ["Progress Bars"],
+                    ["Buttons"],
```
Comment (JaydeepParmar-iView) [inline src/components/NavBar/Hamburger.tsx]: make a variable and then map

File: src/components/NavBar/Hamburger.tsx [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+              <Accordion.Title className="text-black font-normal">
+                Blog
+              </Accordion.Title>
+              <Accordion.Content className="p-0">
+                <ul className="flex flex-col gap-3">
+                  {[
+                    ["Classic Blog"],
+                    ["Grid Blog"],
+                    ["Grid Blog 2"],
+                    ["Grid Blog 3"],
```
Comment (JaydeepParmar-iView) [inline src/components/NavBar/Hamburger.tsx]: make a variable then map

File: src/components/NavBar/NavBar.tsx [hunk 5]
```diff
@@ -0,0 +1,286 @@
...
+          <Link to="/" className="group">
+            <div>Pricing</div> <HoverlineEffect />
+          </Link>
+          <Link
+            to="/"
+            className="group relative h-16 flex flex-col justify-center items-center"
+          >
+            <div>Blog</div> <HoverlineEffect />
+            <Card className="z-10 rounded-none font-semibold text-base w-60 px-4 mt-2 absolute left-0 whitespace-nowrap hidden group-hover:block hover:block group-hover:transform group-hover:top-14 transition-all duration-300 ease-in bg-white">
+              <ul className="flex flex-col gap-3">
```
Comment (JaydeepParmar-iView) [inline src/components/NavBar/NavBar.tsx:80]: if there are same component then please re-use it.

File: src/components/NavBar/NavBar.tsx [hunk 6]
```diff
@@ -0,0 +1,286 @@
+import { useRef, useState } from "react";
+import { Link } from "react-router-dom";
+import { Card, Carousel } from "flowbite-react";
+import logo from "../../assets/logo-dreamsoft.png";
+import NavbarImg1 from "../../assets/navbar-img-1.jpg";
```
Comment (Prashant-021) [inline src/components/NavBar/NavBar.tsx]: Keep names of variables more meaningful.

File: src/components/Slider/Slider.tsx [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+            Slider2,
+            "Experienced Team",
+            `We are a team of qualified software developers, aimed at creating unique and powerful tools for your business & everyday life.`,
+          ],
+          [
+            Slider3,
+            "Award-Winning Software",
+            `The software solutions developed by our company have been numerously awarded for usability and innovative features.`,
+          ],
+        ].map((slider) => (
```
Comment (JaydeepParmar-iView) [inline src/components/Slider/Slider.tsx]: make a variable

**Review by JaydeepParmar-iView — CHANGES_REQUESTED** [review]
Please do changes accordingly.

File: src/components/NavBar/NavBar.tsx [hunk 8]
```diff
@@ -0,0 +1,286 @@
...
+            <div>Pricing</div> <HoverlineEffect />
+          </Link>
+          <Link
+            to="/"
+            className="group relative h-16 flex flex-col justify-center items-center"
+          >
+            <div>Blog</div> <HoverlineEffect />
+            <Card className="z-10 rounded-none font-semibold text-base w-60 px-4 mt-2 absolute left-0 whitespace-nowrap hidden group-hover:block hover:block group-hover:transform group-hover:top-14 transition-all duration-300 ease-in bg-white">
+              <ul className="flex flex-col gap-3">
+                {[
```
Comment (Prashant-021) [inline src/components/NavBar/NavBar.tsx]: Create all arrays as variables at beginning of the file and use it here

**Comment by Prashant-021** [comment]
Create theme for whole website

**Pull request merged**